docs/en/deploy/Running-Alluxio-On-a-HA-Cluster.md

```diff
@@ -434,11 +430,10 @@ Alternatively, one benefit of running Alluxio in HA mode is to use rolling resta
 to minimize downtime when updating configurations:
 
 1. Update the master configuration on all the master nodes without restarting any master.
-1. Restart the leading master (can be determined by running `bin/alluxio leader`).
-  A new leading master will be elected to continue servicing requests.
-1. Wait for the previous leading master to come up successfully but as a standby master.
-1. Update and restart all remaining standby masters.
-1. Verify the configuration update
+2. Restart standby masters one by one (the cluster [cannot survive more than `floor(n/2)` simultaneous restarts](http://localhost:4000/en/operation/Journal.html#embedded-journal-vs-ufs-journal)).
+3. Elect a standby master as the leading master (tutorial [here](http://localhost:4000/en/operation/Journal.html#electing-a-specific-master-as-leader)).
```

Comment:
I'm guessing you meant to use relative links and not the localhost ones?

Reply:
yep, fixed